packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -456,6 +509,8 @@ function getPostOrder(posts: Array<PostsList>, votes: Array<qualitativeVote|quad
       return [post, voteForPost, voteScore, i, randomPermutation[i]]
     })
     .sort(([post1, vote1, voteScore1, i1, permuted1], [post2, vote2, voteScore2, i2, permuted2]) => {
```

Comment:
Could use a sanity check here. Intended behavior is that posts you haven't voted on with at least one review are sorted to the top, followed by posts you've voted on, followed by everything else.

Reply:
Yeah agreed. The thing that was high friction here was that I'd need to go set up a whole suite of review-esque data on the testing db. (or, I guess, make it possible to do easy unit testing of components that I trusted to matter. The "trust to matter" there is doing a lot of work)